diff -Nur -x '*.orig' -x '*.rej' e-smith-pop3-1.2.0/createlinks mezzanine_patched_e-smith-pop3-1.2.0/createlinks
--- e-smith-pop3-1.2.0/createlinks	2005-08-10 16:49:09.000000000 -0400
+++ mezzanine_patched_e-smith-pop3-1.2.0/createlinks	2006-03-24 16:00:48.000000000 -0500
@@ -2,7 +2,8 @@
 
 use esmith::Build::CreateLinks qw(:all);
 
-foreach my $event (qw(bootstrap-console-save network-delete network-create email-update))
+foreach my $event (qw(
+    bootstrap-console-save network-delete network-create email-update))
 {
     foreach (qw(pop3/peers/0 pop3/peers/local pop3s/peers/0 pop3s/peers/local))
     {
@@ -10,9 +11,12 @@
     }
 }
 
-foreach my $event (qw(network-delete network-create))
+foreach my $event (qw(network-delete network-create email-update))
 {
-    safe_symlink("sigusr1", "root/etc/e-smith/events/$event/services2adjust/pop3");
+    safe_symlink("sigusr1",
+	"root/etc/e-smith/events/$event/services2adjust/pop3");
+    safe_symlink("sigusr1",
+	"root/etc/e-smith/events/$event/services2adjust/pop3s");
 }
 
 safe_symlink("daemontools", "root/etc/rc.d/init.d/pop3");
